Code/GraphMol/MolDraw2D/MolDraw2DSVG.cpp

```diff
@@ -259,6 +260,16 @@ void MolDraw2DSVG::getStringSize(const std::string &label, double &label_width,
   }
 }
 
+namespace {
+void escape_xhtml(std::string &data) {
```

Comment:
This code is fine, but there is also:

https://www.tena-sda.org/doc/5.2.1/boost/d3/df1/namespaceboost_1_1spirit_1_1xml.html

And we already use the spirit library.